app/templates/_brief_attributes.html

```diff
@@ -33,7 +33,9 @@
     ) %}
       {% call summary.row() %}
         {{ summary.field_name(item.label) }}
-        {{ summary[item.type](item.value) | format_links}}
+        {% with item_value = summary[item.type](item.value) | format_links %}
+            {{ item_value | preserve_line_breaks if item.type == 'textbox_large' else item_value }}
```

Comment:
Some parenthesis might be nice, if supported, to show that `item_value | preserve_line_breaks` is definitely interpreted as one item.